src/Viking.hs

```diff
@@ -71,4 +71,4 @@ import           Streaming (Stream, Of(..))
 -- @
 --
 type ByteStream =
-  ByteString
+  Streaming.ByteStream
```

Comment:
This is redundant now isn't it? We should be able to just export the base type.

Reply:
OK, I thought the type synonym served a more minor purpose, will change.

Reply:
Essentially, we thought that `ByteString` was a bad name for this type. Eventually the upstream guys agreed.